### PR TITLE
Document troubleshooting with `kernel-next`

### PR DESCRIPTION
**Pull request by strugee**
Fixes QubesOS/qubes-issues#4495

### CODE REVIEW

**Comment by strugee** [comment]
Questions/notes:

* `kernel` _is_ the kernel from Fedora 25, right?
* Is the YAML frontmatter right? I couldn't quite tell
* I couldn't build the site locally since there wasn't any documentation on how to do it, so I haven't really tested how this looks

**Comment by marmarek** [comment]
>     * `kernel` _is_ the kernel from Fedora 25, right?

No, Fedora 25 come with kernel 4.8... We ship LTS kernel in `kernel` package.
BTW 4.19 kernel was announced to be LTS, so after some testing we'll switch `kernel` package to that. `kernel-latest` will still track newer version (4.20/5.0, 4.21/5.1, ...).

> I couldn't build the site locally since there wasn't any documentation on how to do it, so I haven't really tested how this looks

See readme in main repository: https://github.com/QubesOS/qubesos.github.io/. You need main repository to build it.

See also travis log (scroll to the end) - there is a typo: `succesfully`

>     * Is the YAML frontmatter right? I couldn't quite tell

Looks ok.

Thanks!

File: troubleshooting/newer-hardware-troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+layout: doc
+title: Troubleshooting newer hardware
+permalink: /doc/newer-hardware-troubleshooting/
+---
+
+Troubleshooting newer hardware
+==============================
+
+By default, the kernel that is installed in dom0 comes from the `kernel` package, which is an older Linux LTS kernel.
+For most cases this works fine since the Qubes OS developers backport fixes to this kernel, but for some newer hardware, you may run into issues.
```
Comment (marmarek) [inline troubleshooting/newer-hardware-troubleshooting.md]: To be precise, Linux kernel developers does that - see https://www.kernel.org/category/releases.html

File: troubleshooting/newer-hardware-troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+To fix this, you can try the `kernel-latest` package - though be aware that it's less tested!
+In dom0:
+
+~~~
+sudo qubes-dom0-update kernel-latest
+~~~
+
+Reboot when it's done installing.
+You can double-check that the boot used the newer kernel with `uname -r`, which prints the version of the currently-running kernel.
+Compare this with the output of `rpm -qi kernel | grep Version`.
```
Comment (marmarek) [inline troubleshooting/newer-hardware-troubleshooting.md]:
```suggestion
Compare this with the output of `rpm -q kernel`.
```

**Pull request merged**